src/darwin/Framework/CHIP/MTRDevice_Concrete.mm

```diff
@@ -1646,11 +1646,6 @@ - (void)_scheduleClusterDataPersistence
     }
     [_mostRecentReportTimes addObject:[NSDate now]];
 
-    {
-        std::lock_guard lock(_descriptionLock);
-        _mostRecentReportTimeForDescription = [_mostRecentReportTimes lastObject];
```

Comment:
Why is this being removed?

Reply:
I just copied and pasted the base implementation into concrete, please fix :)